Components/MouseAnimator.js

```diff
@@ -5,14 +5,27 @@ import _ from 'lodash';
 export default class MouseAnimator {
   hook = () => {
     const layout = document.getElementsByClassName('Layout')[0];
+    const showMore = document.getElementsByClassName('ShowMore-overlay');
 
     const callback = (event) => {
       const mouseXRatio = event.clientX / window.innerWidth;
+      const currentColor = this.currentColor(mouseXRatio);
 
-      layout.style.setProperty('--color-background', this.currentColor(mouseXRatio));
+      layout.style.setProperty('--color-background', currentColor);
+
+      _.forEach(showMore, (overlay) => {
+        overlay.style.setProperty(
+          'background-image',
+          `linear-gradient(to bottom, rgba(8,9,64,0), ${currentColor})`,
+        );
+      });
     };
 
     layout.addEventListener('mousemove', _.throttle(callback, 50));
+
+    _.forEach(showMore, (overlay) => {
+      overlay.addEventListener('mousemove', _.throttle(callback, 50));
+    });
```

Comment:
what's the point of this? the single callback above will already catch all mouse movement